### DISCUSSION TITLE
Not save data for a trial

### DISCUSSION THREAD

**Discussion by fspinar** [NYUCCL/psiTurk#558] [Q&A] [answered]
Hi all,

I want to skip saving data in a trial if the response is not correct. I.e.,

if response not correct
  data does not change
else
  update data
  
How can I achieve that?
Thanks!

**Comment by jacob-lee** [comment]
I mean, this is really up to you when you code your task in JavaScript, since generally you have to explicitly call to save the date, e.g. psiTurk.savedata() ...

**Reply by fspinar** [reply]
Hi @jacob-lee 
Thanks for the answer. What I meant was not actually saving the data, but rather not updating it at all.
I mean, if in a given trial the response is null, then I do not want to include that trial in the "data" structure of jsPsych. 
Can I achieve this by calling psiTurk.savedata() appropriately?

Thanks!

**Comment by jacob-lee** [comment, marked as answer]
psiTurk does not have a jsPsych dependency.

Call psiTurk.recordTrialData with just the data that you want to be
eventually saved (by saveData), and no other.

**Reply by fspinar** [reply]
Thanks @jacob-lee
